cdserver/features/monitoring_and_control/step_definitions/monitoring_and_control_steps.rb

```diff
@@ -14,7 +14,8 @@
 
 Then(/^I should see the log '(.*?)'$/) do |expected_message|
   statsd_log =  File.readlines('log/statsd.log')
-  last_log = statsd_log.last
+  expect(statsd_log.count).to eq(@existing_log_count + 1)
```

Comment:
please capture it like this:
Then I should see the log
| Level | Message |
| INFO | User.login.success |